### PR TITLE
Update knowledge check content

### PR DESCRIPTION
**Pull request by thatblindgeye**
## Issue

Closes #XXXXX

## Pull Request Requirements

-   [ ] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [ ] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [ ] The `Because` section summarizes the reason for this PR
-   [ ] The `This PR` section has a bullet point list describing the changes in this PR
-   [ ] If this PR addresses an open issue, it is linked in the `Issue` section
-   [ ] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [ ] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

**Comment by CouchofTomato** [comment]
@thatblindgeye You need this reviewed?

**Comment by thatblindgeye** [comment]
@CouchofTomato yep, if you'd want to feel free, is based on conversation on Discord. Otherwise I've tagged Carlos to review.

**Review by bycdiaz — APPROVED** [review]
I love it. 😭

**Comment by JustWaveThings** [comment]
Small typo on line 73:

> The following questions are on opportunity to reflect... 

Should be:
>The following questions are an opportunity to reflect...

File: LAYOUT_STYLE_GUIDE.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ CUSTOM SECTION CONTENT.
 
 ### Knowledge check
 
-This section contains questions for you to check your understanding of this lesson on your own. If you’re having trouble answering a question, click it and review the material it links to.
+The following questions are on opportunity to reflect on key topics in this lesson. If you can't answer a question, click on it to review the material, but keep in mind you are not expected to memorize or master this knowledge.
```
Comment (thatblindgeye) [inline LAYOUT_STYLE_GUIDE.md:73]:
```suggestion
The following questions are an opportunity to reflect on key topics in this lesson. If you can't answer a question, click on it to review the material, but keep in mind you are not expected to memorize or master this knowledge.
```

**Comment by thatblindgeye** [comment]
@JustWaveThings thanks for catching that!

**Pull request merged**